pipeline-runner/R/gem2s-2-load_user_files.R

```diff
@@ -239,6 +209,38 @@ parse_rhapsody_matrix <- function(config, input_dir) {
   return(list(counts_list = counts_list, annot = annot))
 }
 
+read_10x_annotations <- function(annot_fpath, sample) {
```

Comment:
while I more or less understand the steps inside this function, I'm having a hard time understanding the overall goal of the function. Maybe a short description would be helpful. Specially about what's the final desired format of the annotations.